corehq/apps/app_manager/views/apps.py

```diff
@@ -404,6 +407,21 @@ def app_from_template(request, domain, slug):
     app = import_app_util(template, domain, {
         'created_from_template': '%s' % slug,
     })
+
+    for path in get_template_app_multimedia_paths(slug):
+        media_class = None
+        with open(os.path.join(app_template_dir(slug), path), "rb") as f:
+            f.seek(0)
+            data = f.read()
+            media_class = CommCareMultimedia.get_class_by_data(data)
+            if media_class:
+                multimedia = media_class.get_by_data(data)
+                multimedia.attach_data(data,
+                                       original_filename=path.split(os.sep)[-1],
```

Comment:
nit: there is a function for that: `os.path.basename(path)`

Reply:
thanks, fixed